sycl/doc/extensions/PropertyList/SYCL_EXT_ONEAPI_property_list.asciidoc

```diff
@@ -626,75 +596,73 @@ static_assert(std::is_same_v<t3, bool>);
 Properties should provide semantically meaningful functions to access the parameters' values.
 ====
 
-For functions that take a property list parameter, it is possible to restrict
+For functions that take a `properties` as a parameter, it is possible to restrict
```

Comment:
Nit: I think this was clearer as "a property list parameter", but mainly because I don't really like "a `properties`".